pkg/ast/spl/spl.peg

```diff
@@ -1360,7 +1360,7 @@ BinOptionAlignTime <- "aligntime" EQUAL utcEpoch:(PositiveIntegerAsString) {
 
     return binOptionArgs, nil
 }
-/ "aligntime" EQUAL timestamp:(AbsoluteTimestamp / RelativeTimestamp) {
+/ "aligntime" EQUAL timestamp:(AbsoluteTimestamp / RelativeTimestamp / ChainedRelativeTimestamps) {
```

Comment:
I don't see `ChainedRelativeTimestamps` defined. I see only `ChainedRelativeTimestamp`. Is this just a spelling mistake? 
Also pigeon did not throw this as an error? Since it is not defined?

Reply:
Same comment for all other palces.

Reply:
Addressed it in the latest commit.